composer.json

```diff
@@ -32,7 +32,8 @@
         "zendframework/zend-view": "~2.5",
         "zendframework/zend-http": "~2.5",
         "zendframework/zend-paginator": "~2.5",
-        "zendframework/zend-cache": "~2.5"
+        "zendframework/zend-cache": "~2.5",
+        "zendframework/zend-text": "~2.5"
```

Comment:
Move this to "suggest" and optional to "require-dev"

since not everybody use the console renderer, its not required...In theory we could also move zend-cache when no export is used...but thats a seperate PR